### PR TITLE
[chore]: Prepare for v0.68.0 release

### PR DESCRIPTION
**Pull request by PaurushGarg**
PR to prepare for release v0.68.0. 
Issue reference: #1362

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
 
 # Represents the current release of Java instrumentation.
 # Should match autoinstrumentation/java/version.txt
-autoinstrumentation-java=1.21.0
+autoinstrumentation-java=1.22.1
 
 # Represents the current release of NodeJS instrumentation.
 # Should match value in autoinstrumentation/nodejs/package.json
-autoinstrumentation-nodejs=0.34.0
+autoinstrumentation-nodejs=0.35.0
```
Comment (Aneurysm9) [inline versions.txt:19]:
```
    logger.go:42: 20:15:15 | instrumentation-nodejs-multicontainer | 2023-01-19 20:12:50 +0000 UTC	Warning	Pod my-deployment-with-sidecar-5f4d86f6d5-h5hfw.spec.initContainers{opentelemetry-auto-instrumentation}		Failed	Failed to pull image "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0": rpc error: code = NotFound desc = failed to pull and unpack image "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0": failed to resolve reference "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0": ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0: not found	
```

[`v0.34.0`](https://github.com/open-telemetry/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fautoinstrumentation-nodejs) appears to be the most recent version avilable here.

Comment (PaurushGarg) [reply versions.txt]: Thanks. I have updated now.

**Review by pavolloffay — APPROVED** [review]
@PaurushGarg thanks for the PR ❤️ 

are you planning to submit release PRs to operators repos as well? 

form `release.md`:
```
After the release, generate a new OLM bundle (make bundle) and create two PRs against the Community Operators repositories:
one for the community-operators-prod, used by OLM on Kubernetes. Example: https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/494
one for the community-operators directory, used by Operatorhub.io. Example: https://github.com/k8s-operatorhub/community-operators/pull/461
```

**Comment by pavolloffay** [comment]
@jaronoff97 could you please review as well?

**Review by jaronoff97 — APPROVED** [review]
looks great, glad the chloggen is releasing as expected :)

**Review by Aneurysm9 — APPROVED** [review]

**Comment by PaurushGarg** [comment]
> @PaurushGarg thanks for the PR ❤️
> 
> are you planning to submit release PRs to operators repos as well?
> 
> form `release.md`:
> 
> ```
> After the release, generate a new OLM bundle (make bundle) and create two PRs against the Community Operators repositories:
> one for the community-operators-prod, used by OLM on Kubernetes. Example: https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/494
> one for the community-operators directory, used by Operatorhub.io. Example: https://github.com/k8s-operatorhub/community-operators/pull/461
> ```

@pavolloffay I would like to request for someone else to do this part. I am having issues on my localbox to resolve some dependecies using `go mod tidy`. I think some of these dependecies do not work when `GOPROXY` is set to `direct`. Hence I am not able to run `make bundle `commands as well on my local.

**Comment by iblancasa** [comment]
@PaurushGarg If you want, I can help with this

**Review by frzifus — APPROVED** [review]

**Review by iblancasa — APPROVED** [review]

**Comment by PaurushGarg** [comment]
> @PaurushGarg If you want, I can help with this

@iblancasa that would be great. Thanks a lot.

**Comment by iblancasa** [comment]
I'll do as soon as this PR is merged

**Pull request merged**